client/src/components/settings/keyboard-shortcuts.tsx

```diff
@@ -17,8 +16,7 @@ export default function KeyboardShortcutsSettings({
   const { t } = useTranslation();
 
   return (
-    <Form
-      inline={true}
+    <form
```

Comment:
Are there any reasons to keep this form wrapper? a11y, maybe? It doesn't seem to be doing anything.

Reply:
Sure thing. Let's keep this focused!